PowerShell/JumpCloud Commands Gallery/Windows Commands/Windows - Install Remote Assist.md

```diff
@@ -0,0 +1,52 @@
+#### Name
+
+Windows - Install Remote Assist | v1.0 JCCG
+
+#### commandType
+
+windows
+
+#### Command
+
+```
+$installerURL="https://cdn.awsstg.jumpcloud.com/TheJumpCloud/jumpcloud-remote-assist-agent/latest/jumpcloud-assist-app.exe"
+
+############### Do Not Edit Below This Line ###############
+
+$installerTempLocation="C:\Windows\Temp\JumpCloudRemoteAssistInstaller.exe"
+
+Write-Host "Downloading JumpCloud Remote Assist installer now."
+try {
+    Invoke-WebRequest -Uri $installerURL -OutFile $installerTempLocation
+}
+catch {
+    Write-Error "Unable to download JumpCloud Remote Assist installer."
+    exit 1
+}
+Write-Host "Finished downloading JumpCloud Remote Assist installer."
+
+Write-Host "Installing JumpCloud Remote Assist now, this may take a few minutes."
+try {
+    $installerProcess = Start-Process -FilePath $installerTempLocation -Wait
+}
+catch {
+    Write-Error "Failed to run Falcon Agent installer."
+    exit 1
+}
+Write-Host "JumpCloud Remote Assist installer returned $($installerProcess.ExitCode)."
+
+exit $installerProcess.ExitCode
+
+```
+
+#### Description
+
+This command will download and install the JumpCloud Remote Assist app on a Windows device.
+
+### *Import This Command*
+
+To import this command into your JumpCloud tenant run the below command using the [JumpCloud PowerShell Module](https://github.com/TheJumpCloud/support/wiki/Installing-the-JumpCloud-PowerShell-Module)
+
+```
+Import-JCCommand -URL 'TODO'
```

Comment:
https://git.io no longer accepting links to shorten :/ I think we want ...

`Import-JCCommand -URL 'https://github.com/TheJumpCloud/support/blob/master/PowerShell/JumpCloud%20Commands%20Gallery/Windows%20Commands/Windows%20-%20Install%20Remote%20Assist.md'`